sbncode/CAFMaker/FillReco.cxx

```diff
@@ -273,6 +283,38 @@ namespace caf
       else {
         srslice.nu_score = -1;
       }
+      // NeutrinoID (SliceID) features
+      if (properties.count("NuNFinalStatePfos")) {
+        srslice.nuid.NuNFinalStatePfos = properties.at("NuNFinalStatePfos");
+      }
```

Comment:
I wonder if we can make some `CopyPropertyIfSet()` helper function to reduce the repetition here.
```suggestion
      CopyPropertyIfSet(properties, "NuNFinalStatePfos", srslice.nuid.NuNFinalStatePfos);
```